### ISSUE TITLE
`gcc` has started to print a lot of "warning: "__NR_bpf" redefined" messages on Ubuntu Xenial

### ISSUE THREAD

**Issue by evverx** [systemd/systemd#8238]
I first saw these messages at https://objectstorage.prodstack4-5.canonical.com/v1/AUTH_77e2ada1e7a84929a74ba3b87153c0ac/autopkgtest-xenial-pitti-systemd-semaphore/xenial/amd64/s/systemd-upstream/20180220_235029_3fd8d@/log.gz
```sh
'src/basic/basic@sta/bitmap.c.o' -c ../src/basic/bitmap.c
In file included from ../src/basic/missing.h:1416:0,
                 from ../src/basic/util.h:47,
                 from ../src/basic/hashmap.h:30,
                 from ../src/basic/bitmap.h:25,
                 from ../src/basic/bitmap.c:28:
../src/basic/missing_syscall.h:392:0: warning: "__NR_bpf" redefined
 #      define __NR_bpf 332
 ^
In file included from /usr/include/x86_64-linux-gnu/asm/unistd.h:12:0,
                 from /usr/include/x86_64-linux-gnu/sys/syscall.h:24,
                 from ../src/basic/missing.h:43,
                 from ../src/basic/util.h:47,
                 from ../src/basic/hashmap.h:30,
                 from ../src/basic/bitmap.h:25,
                 from ../src/basic/bitmap.c:28:
/usr/include/x86_64-linux-gnu/asm/unistd_64.h:325:0: note: this is the location of the previous definition
 #define __NR_bpf 321
 ^
```
It seems that in 4c2e1b399f5809 `__NR_statx` should have been used instead of `__NR_bpf`.

**Comment by poettering** [comment]
Ouch!

**Comment by poettering** [comment]
Fix waiting in #8243
